### DISCUSSION TITLE
There are no commands defined in the "flare" namespace.

### DISCUSSION THREAD

**Discussion by deep-codepaper** [spatie/laravel-ignition#160] [Q&A]
i tried both 2.0 and 2.3 version but im still getting same issue on ubuntu:22.04. but still getting There are no commands defined in the "flare" namespace when run php artisan flare:test

**Comment by deep-codepaper** [comment]
no answer

**Comment by deep-codepaper** [comment]
@freekmurze

**Comment by freekmurze** [comment]
The command is registered here: https://github.com/spatie/laravel-ignition/blob/5e0dfd32b87d7f6c1623629dfa0935828f78349f/src/IgnitionServiceProvider.php#L79 and the command itself does exist. It works in all projects and it seems we're not getting any similar reports.

If you find out why it's not working for you, and know the fix, we'd happily accept a PR.

**Comment by benjibee** [comment]
@deep-codepaper I was having the same issue on production, but not on my local environment. After removing and reinstalling the package, trying lots of other solutions I had a look at the above service provider and saw that the only check that the `registerCommands()` method does is looking for the Flare environment variable:

```php
protected function registerCommands(): void
{
    if ($this->app['config']->get('flare.key')) {
        $this->commands([
            TestCommand::class,
        ]);
    }

    […]
}
```

Sure enough, I had a typo in the environment variable on production, and changing it to `FLARE_KEY` solved the issue directly.
